src/main/java/core/basesyntax/Behaviour.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Behaviour {
```

Comment:
```suggestion
public interface AreaCalculator {
```
behaviour is general term to describe what object can do)